campair/campair/View/Editor/Detail/OpeningEditorView.swift

```diff
@@ -12,32 +12,39 @@ struct OpeningEditorView: View {
     var body: some View {
         VStack(alignment: .leading, spacing: 0) {
             ZStack(alignment: .topLeading) {
-                Image("examplePainting")
-                    .resizable()
-                    .frame(height: 422)
-                VStack(alignment: .leading, spacing: 10) {
+                VStack(alignment: .leading, spacing: 0) {
                     Text("\(self.openingSection.cardTitle)")
-                        .font(.system(size: 15))
-                        .bold()
-                        .foregroundColor(Color.customPaintingTextPink)
+                        .font(.system(.subheadline))
+                        .fontWeight(.regular)
+                        .foregroundColor(Color(hex: self.openingSection.cardPaintingTextColor))
+                        .padding(.bottom, 6)
                     Text("\(self.openingSection.cardSubTitle)")
+                        .font(.system(.title2))
+                        .fontWeight(.bold)
                         .foregroundColor(Color.white)
-                        .font(.system(size: 28))
-                        .fontWeight(.heavy)
+                Image("contentCard2")
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: 350, height: 300)
                 }
-                .padding(.top, 30 + UIDevice.current.getSafeAreaTopValue)
-                .padding(.top, 55)
-                .padding(.leading, 20)
+                .padding(.horizontal, 20)
+                .padding(.bottom, 20)
+                .padding(.top, 34 + UIDevice.current.getSafeAreaTopValue)
             }
-            VStack(alignment: .leading, spacing: 11) {
+            .background(Color(hex: self.openingSection.cardPaintingBackgroundColor))
+            VStack(alignment: .leading, spacing: 10) {
                 Text("\(self.openingSection.title)")
-                    .foregroundColor(Color.customTextPink)
+                    .foregroundColor(Color(hex: self.openingSection.cardTitleColor))
+                    .font(.system(.headline, design: .default))
                 Text("\(self.openingSection.subTitle)")
-                    .foregroundColor(Color.customTextGray)
+                    .foregroundColor(Color.customDarkGray)
+                    .font(.system(.body, design: .default))
+                // 스케치에서는 26으로 되어있는데 폰트 사이즈를 빼서 계산을 해야 한다고... but 쉬운 방법이 있을 거 같아서 찾아보는 중입니다.
```

Comment:
```suggestion
                // TODO: 스케치에서는 26으로 되어있는데 폰트 사이즈를 빼서 계산을 해야 한다고... but 쉬운 방법이 있을 거 같아서 찾아보는 중입니다.
```

수정해야 할 부분은 TODO가 좋겠지요 .. !

Reply:
감사합니다 ~